lib/client/ipam_block_reader_writer.go

```diff
@@ -30,6 +30,10 @@ import (
 	cnet "github.com/projectcalico/libcalico-go/lib/net"
 )
 
+// affinedTo variable represents what the resource (host in
+// this case) is affined to.
+var affinedTo = "host"
```

Comment:
Don't think there's much point in having this defined like this -  when we do add more types of affinity, it's going to be way more significant a change than just changing the value of a constant.

Let's just inline the "host" string.